audit-filter/README.md

```diff
@@ -0,0 +1,85 @@
+# audit-log-exporter
+Enables policy-driven event filtering and deduplication of OpenShift audit logs for splunk-forwarder
+
+## Enabling in stage
+
+Forwarding audit logs to `openshift_managed_audit_stage` can be enabled by adding a policy file to the `osd-monitored-logs-local` configmap:
+
+```shell
+oc set data cm/osd-monitored-logs-local --from-file=policy.yaml
+```
```

Comment:
Unclear whether the [supplied policy.yaml file](https://github.com/openshift/boilerplate/blob/master/README.md#update) is intended for this purpose; and if it is, whether it must/should be edited first.